README.md

```diff
@@ -19,60 +19,31 @@ This is an ASP.NET Core Web API reference application designed to "fork and code
 
 ## Prerequisites
 
-- Bash shell (tested on Mac, Ubuntu, Windows with WSL2)
-  - Will not work with WSL1
-  - Docker commands will not work in Cloud Shell
+- Bash shell (tested on Mac, Ubuntu, Windows with WSL2, CloudSpaces)
+  - Will not work with WSL1 or Cloud Shell
```

Comment:
Should this be Codespaces (and throughout readme)? Or is this a more general term?

Reply:
Oops ... fixed